RR.App/Controllers/HRIS/PropertyAccessController.cs

```diff
@@ -105,21 +100,14 @@ public async Task<IActionResult> GetUserId(string email)
     {
         try
         {
-            var claimsIdentity = User.Identity as ClaimsIdentity;
-            var accessTokenEmail = claimsIdentity?.FindFirst(ClaimTypes.Email)?.Value;
-            var role = claimsIdentity?.FindFirst(ClaimTypes.Role)?.Value;
-
-            if (!string.IsNullOrEmpty(accessTokenEmail))
+            if (!string.IsNullOrEmpty(_identity.Email))
             {
-                if (email == accessTokenEmail)
+                if (email == _identity.Email)
                 {
-                    var employee = await _employeeService.GetEmployee(email);
-                    //TODO: find a better way to add auth0 user id to the db, perhaps employee id and auth id should be the same?
-                    GlobalVariables.SetUserId(employee!.Id);
```

Comment:
Logic: this setUserId is done here when the user already exists. Since the CheckUser method will be skipped on the front end. The id is pulled from the database.